shared/map/TileType.js

```diff
@@ -1,23 +1,24 @@
-import { isDistrict, isPlaza, isRoad } from '../shared/district'
```

Comment:
Both this file and `TileLocation` work (mostly) with a parcel or a tile. This is because the interface is really similar. Because we don't have typescript it's kind of hard to communicate, I don't have a name/abstraction for a `parcel/tile/interface`

It's on my radar